src/wallet/rpcwallet.cpp

```diff
@@ -2103,8 +2180,8 @@ UniValue walletpassphrasechange(const UniValue& params, bool fHelp)
 
     if (strOldWalletPass.length() < 1 || strNewWalletPass.length() < 1)
         throw runtime_error(
-            "walletpassphrasechange <oldpassphrase> <newpassphrase>\n"
-            "Changes the wallet passphrase from <oldpassphrase> to <newpassphrase>.");
+            R"(walletpassphrasechange <oldpassphrase> <newpassphrase>
+            Changes the wallet passphrase from <oldpassphrase> to <newpassphrase>.)");

```

Comment:
trim whitespace from the left, otherwise second line will be shifted

Reply:
done thanks